app/models/model.rb

```diff
@@ -0,0 +1,6 @@
+class Model < ApplicationRecord
```

Comment:
You actually created a model named model! I am bit worried now...

Time to read <https://github.com/plataformatec/devise#getting-started> again